app/serializers/api/v3/statement_serializer.rb

```diff
@@ -0,0 +1,18 @@
+module API
+  module V3
+    class StatementSerializer < Blueprinter::Base
+      identifier :id
+
+      field :month
+      field :year
+      field :cohort_id, name: :cohort
+      field :deadline_date, name: :cut_off_date, datetime_format: "%Y-%m-%d"
+      field :payment_date, datetime_format: "%Y-%m-%d"
+      field :created_at, datetime_format: ->(dt) { dt.iso8601 }
+      field :updated_at, datetime_format: ->(dt) { dt.iso8601 }
```

Comment:
Probably makes sense to just set a [default format for BluePrinter](https://github.com/procore-oss/blueprinter?tab=readme-ov-file#global-config-setting---datetime) here and use `iso8601`, overriding if necessary.

Looking at these examples though it'll give us what we want without needing anything extra to be defined.

```ruby
DateTime.now.iso8601
=> "2024-03-07T10:09:26+00:00"

DateTime.now.strftime("%Y-%m-%d")
=> "2024-03-07"

Date.today.iso8691
=> "2024-03-07"
```